### DISCUSSION TITLE
How to add documents to firestore with vuefire (Nuxt 3)

### DISCUSSION THREAD

**Discussion by svirmasalo** [vuejs/vuefire#1339] [Questions and Help] [answered]
Hi,

I've been trying to figure out how to add a document into a collection with Vuefire, but can't seem to find the way to do this. 

Could someone point me into a right direction?

Thanks!

**Comment by posva** [comment, marked as answer]
You use the Firebase JS SDK for that with its methods like `addDoc()` and `setDoc()`
